### DISCUSSION TITLE
Forced Discharge with Foxess Inverter

### DISCUSSION THREAD

**Discussion by Weiss68** [macxq/foxess-ha#136] [Q&A]
I can't find any info online or in the cloud management page for this so I figured someone one here might know.

I can force charge my batteries and have been during the winter on cheaper night rates but I have now swapped to Octopus Flux which has a night rate import of 10p and a 3 hour evening export of 35p. On a normal day (April - Sept) my panels will be exporting during this evening window but in the interests of making money I'd like to empty 60-70% of my battery during this time and then top it up (still debating this as I may not need the top up) at night on the cheap rate.

Has anyone here found a way to force discharge in the same way you can force Charge the batteries in defined time windows?

For reference I have a H1-6.0-E inverter and 10kW fox batteries.

Cheers

**Comment by Thebig24fan** [comment]
No not automatic from fox as far as i know but yes it is possible to export from the battery’s but you need do it yourself or if my idea from other people's info works might be able to setup force discharge at a set time automatically with a bit of electrical knowledge

From the desk of Nickie
________________________________
From: Weiss68 ***@***.***>
Sent: Friday, March 31, 2023 9:02:31 AM
To: macxq/foxess-ha ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [macxq/foxess-ha] Forced Discharge with Foxess Inverter (Discussion #136)


I can't find any info online or in the cloud management page for this so I figured someone one here might know.

I can force charge my batteries and have been during the winter on cheaper night rates but I have now swapped to Octopus Flux which has a night rate import of 10p and a 3 hour evening export of 35p. On a normal day (April - Sept) my panels will be exporting during this evening window but in the interests of making money I'd like to empty 60-70% of my battery during this time and then top it up (still debating this as I may not need the top up) at night on the cheap rate.

Has anyone here found a way to force discharge in the same way you can force Charge the batteries in defined time windows?

For reference I have a H1-6.0-E inverter and 10kW fox batteries.

Cheers
